au/code/au/conversion_policy.hh

```diff
@@ -27,11 +27,11 @@ namespace au {
 template <typename Rep, typename... BPs>
 constexpr bool can_scale_without_overflow(Magnitude<BPs...> m, Rep value) {
     // Scales that shrink don't cause overflow.
-    if (get_value<double>(m) <= 1.0) {
+    if (get_value<double>(abs(m)) <= 1.0) {
         (void)value;
         return true;
     } else {
```

Comment:
Is the `else` redundant here?

```suggestion
        return true;
    }
```

Personally, the sooner I can drop the conditional context from what I've got to keep active in my "internal/mental model" stack when reading code, the easier it is parse.

Reply:
This whole setup feels a little goofy.  I decided to rewrite it as a simple `||` conditional.